bin/main.rb

```diff
@@ -1,80 +1,62 @@
 #!/usr/bin/env ruby
-puts 'Hello World!'
-class Test
-  attr_accessor :board, :input, :x, :o
+class Game
+  require_relative '../lib/board'
+  require_relative '../lib/players'
 
-  def initialize
-    @x = x
-    @o = o
-    @input = input
-    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
-  end
+  attr_accessor :curr_input, :curr_player
```

Comment:
- [ ] `:curr_input` isn't being either read nor written by anything outside the class, so it's not needed. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))